README.md

```diff
@@ -347,8 +347,9 @@ execute(function(finished) {
 });
 
```

Comment:
Need to add a new section in the README to document the suite/tests API you just added.